chef/Vagrantfile

```diff
@@ -0,0 +1,43 @@
+# This Vagrantfile can be used to quickly spin up a development instance of ub1404
```

Comment:
Reconsider filename/path, as is `chef/Vagrantfile` will become confusing quickly if any other VM in this project were to start consuming `chef`.